tests/test_parse_response.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html)
+
+from cubiscan.cubiscan import CubiScan
+
+
+def test_parse_positive_measure_response():
+    scanner_obj = CubiScan('127.0.0.1', '1234')
+    response_msg = bytes('\x02MAC004600,L020.0,W010.0,H030.0,'
+                         'M,K500.50,D510.50,M,F0010,I\x03\x0D\x0A', 'ascii')
+    response = scanner_obj._parse_response('continous_measure', response_msg)
```

Comment:
add a global comment saying that you test only parsing of response. No test on connection